docs/developmentTips.md

```diff
@@ -0,0 +1,10 @@
+
+# Development Tips
+
+## Disabling Persisted State
+
+To ignore the persisted state (i.e. don't auto-login), comment out this line:
+
+https://github.com/ParabolInc/action/blob/master/src/client/makeStore.js#L51
```

Comment:
let's use a commit-specific blob instead of master blob so this stays relevant longer than 1 commit :smile: